scipy/meson.build

```diff
@@ -563,22 +563,36 @@ lapack_dep = declare_dependency(
   dependencies: lapack_dep,
   link_with: [g77_abi_wrappers, blas_lapack_wrapper_lib]
 )
-subdir('_lib')
-subdir('special')
-subdir('linalg')
-subdir('sparse')
-subdir('stats')
-subdir('fft')
-subdir('io')
-subdir('spatial')
-subdir('cluster')
-subdir('constants')
-subdir('fftpack')
-subdir('integrate')
-subdir('signal')
-subdir('interpolate')
-subdir('ndimage')
-subdir('odr')
-subdir('optimize')
-subdir('datasets')
-subdir('misc')
+
+scipy_subdirs = [
```

Comment:
The changes in this file should not be necessary. Tags are already corrected in all the other files, so `subdir` calls are fine. This works differently from NumPy, which uses `install_subdir` instead of `subdir` on sub-directories containing only pure Python code.

Can you please revert all changes to this file?

Reply:
I see. Yes. I will update tomorrow.